intermediate_source/forward_ad_usage.py

```diff
@@ -172,5 +172,75 @@ def jvp(ctx, gI):
                          check_backward_ad=False, check_undefined_grad=False,
                          check_batched_grad=False)
 
+######################################################################
+# Functional API
```

Comment:
Should users be concerned about these APIs changing?

Reply:
The APIs are technically "beta" but I don't see us changing the API in a BC-breaking way at all because that's what JAX offers. I can put "(beta)" next to it to make it clearer